cli/cli.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+"""
+CLI for interfacing with deployed analysis runner.
+See README.md for more information.
+"""
+import re
+import logging
+import subprocess
+
+import click
+import requests
+import google.auth
+import google.auth.transport.requests
+
+logging.basicConfig(level='INFO')
+
+BRANCH = 'main'
+DEFAULT_SERVER_LOOKUP = (
+    f'https://raw.githubusercontent.com/'
+    f'populationgenomics/analysis-runner/{BRANCH}/cli/servermap.json'
+)
+# Make this None if you want to support all organisations
```

Comment:
Can we use the American spelling? (GitHub and Google use organization.)

Reply:
Murica'd it up in: 773fcf724bbbc3f29637ab9326d9a6e4675a2462